acceptance/bundle/deploy/pipeline/recreate/output.txt

```diff
@@ -30,9 +30,10 @@ Deployment complete!
   }
 }
 
+  recreate pipelines.foo
+  recreate schemas.bar
```

Comment:
I was puzzled why the ordering seems off, but it's because of a `| sort` after the trace.

Is that still needed?